frontend/components/card/discussion/CardDiscussionInput.vue

```diff
@@ -97,7 +97,7 @@
         <textarea
           id="message"
           rows="4"
-          class="block p-2.5 w-full text-sm text-light-text rounded-lg border border-light-section-div placeholder-light-distinct-text dark:placeholder-dark-distinct-text dark:text-dark-text dark:bg-dark-distinct focus-brand"
+          class="block p-2.5 w-full text-sm text-light-text rounded-lg border border-light-section-div placeholder-light-distinct-text dark:placeholder-dark-distinct-text dark:text-dark-text dark:bg-dark-layer-1 focus-brand"
```

Comment:
Looks like this is missing light mode bg and a dark mode border. I can get this 😊

Reply:
Oh right, good side effects of going through basically the whole code :P